Crashlytics/Crashlytics/Public/FirebaseCrashlytics/FIRCrashlytics.h

```diff
@@ -16,7 +16,7 @@
 
 #import "FIRCrashlyticsReport.h"
 #import "FIRExceptionModel.h"
-
+@protocol FIRRolloutsStateSubscriber;
```

Comment:
Yes, this can be imported. 

Here's an example where AppCheck imports the [AppCheckInterop](https://github.com/firebase/firebase-ios-sdk/blob/2e1809d53c917cea34c48d37b8f32dba375b13e0/FirebaseAppCheck/Sources/Public/FirebaseAppCheck/FIRAppCheck.h#L19C34-L19C57) header in order to [declare conformance](https://github.com/firebase/firebase-ios-sdk/blob/2e1809d53c917cea34c48d37b8f32dba375b13e0/FirebaseAppCheck/Sources/Public/FirebaseAppCheck/FIRAppCheck.h#L41C36-L41C55) with the `FIRAppCheckProtocol` protocol from the interop SDK.

Reply:
I think it _needs_ to be imported via https://stackoverflow.com/a/11533804/9331576. Were you able to get things building in its current state?

Reply:
Interesting, I see in codebase that we have some pattern for forward declaring a protocol, for example FireAuth: https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseAuth/Sources/Backend/FIRAuthRequestConfiguration.h#L24. I am able to get things build for now. But I'll change to import if it's safer.